spaceship/lib/spaceship/tunes/tunes_client.rb

```diff
@@ -575,12 +575,22 @@ def candidate_builds(app_id, version_id)
     #####################################################
 
     # @param (testing_type) internal or external
-    def build_trains(app_id, testing_type, platform: nil)
+    def build_trains(app_id, testing_type, platform: nil, tries: 5)
```

Comment:
The `tries` value isn't working yet, it seems like it's not used because we're using `:` instead of the `=` notation. Looking into this